openshift/scripts/oc_deploy.sh

```diff
@@ -15,7 +15,7 @@ source "$(dirname ${0})/common/common"
 #%
```

Comment:
Why did the ACL change for this file?
100755 → 100644

Reply:
Good catch - I was moving the file around and it must have changed! No worries, I'll `chmod +x` for the next push.